src/client-services/github.client-service.test.tsx

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import {
+  axiosConfig,
+  getRepoInfo,
+  getReposList,
+} from '../client-services/github.client-service';
+
+describe('Mock Axios', () => {
+  let axiosMock: MockAdapter;
+  const accessToken = process.env.GITHUB_API_KEY;
+
+  // Set new axios mock before each test
+  beforeEach(() => {
+    axiosMock = new MockAdapter(axios);
+  });
+
+  // Remove axios mock after each test
+  afterEach(() => {
+    axiosMock.restore();
+  });
+
+  it('should return an Axios instance', async () => {
```

Comment:
Weird mix of async/await and promise .then

The more modern way to do this is

async () => {
    await somethingAsync();
}